.github/workflows/tests.yml

```diff
@@ -23,9 +23,10 @@ jobs:
       - name: Check out code into the Go module directory
         uses: actions/checkout@v3.0.2
 
-      - name: Check that source has been prepared
+      - name: Check that source has been prepared and formatted
         run: |
           ./tools/prepare-source.sh
+          ./tools/format-source.sh
```

Comment:
I don't want a separate script. It's nice that `tools/prepare-source.sh` does everything needed pre-commit related to formatting, modules and vendoring. We have this script everywhere, so creating a separate workflow in `image-builder` is confusing.

Reply:
Thing is, if you want to reformat (fix) all the code in case CI complains, you would need to run the prepare-source.sh which is quite slow.

Reply:
3 seconds cold start, 2 seconds hot start on my laptop. Doesn't feel that bad. :shrug: But I agree that this is very subjective.